src/components/Cards/NewPost/NewPostCard.jsx

```diff
@@ -1,17 +1,38 @@
-import { avatar } from "../../../assets"
 import { FaRegImages } from "react-icons/fa"
 import { AiOutlineFileGif } from "react-icons/ai"
 import { VscSmiley } from "react-icons/vsc"
+import { useDispatch, useSelector } from "react-redux"
+import { useState } from "react"
+import { createNewPost } from "../../../features/Post/PostSlice"
+import toast from "react-hot-toast"
 
 export const NewPostCard = () => {
+
+    const { userData, authToken } = useSelector(state => state.auth);
+    const dispatch = useDispatch();
+    const [postData, setPostData] = useState({
+        content: "",
+    })
+
+    const postSubmitHandler = (event) => {
+        event.preventDefault();
+        if (postData.content !== "") {
```

Comment:
Suggestions: Can also add a condition for not taking space/blank character as an input